src/editor/components/scenegraph/SceneGraph.js

```diff
@@ -101,6 +103,7 @@ export default class SceneGraph extends React.Component {
         if (this.props.onChange) {
           this.props.onChange(entity);
         }
+        posthog.capture('entity_selected', { entity: entity.tagName });
```

Comment:
Do you want to make a distinction between selecting an entity in scene graph and in the scene (entityselect is also emitted there)?
If you don't care and you want to also track when you select in the scene, you should add in `Main.js` the following:
```js
Events.on('entityselect', (entity) => {
  posthog.capture('entity_selected', { entity: entity.tagName });
})
```

but `entity.tagName` won't me much of interest here, it will always be `A-ENTITY`, you probably want `entity.getAttribute('mixin')` to know the type of asset, so
```js
  posthog.capture('entity_selected', { mixin: entity.getAttribute('mixin') });
```

Reply:
i want to record when entities are selected in the scene graph for now. Maybe down the line it will be interesting to see the other one.